doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -736,6 +736,24 @@ The fully expanded example above (without environment variables) looks like this
 
   curl -H X-Dataverse-key:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx https://demo.dataverse.org/api/dataverses/root/guestbookResponses?guestbookId=1 -o myResponses.csv
 
+.. _collection-attributes-api:
+  
+Change Collection Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: 
+
+  curl -H  X-Dataverse-key:$API_TOKEN $SERVER_URL/api/dataverses/$ID/attribute/$ATTRIBUTE?value=$VALUE
+
+The following attributes are supported:
+
+* ``alias``  Collection alias
+* ``name`` Name
```

Comment:
This should be useful for @Kris-LIBIS.

In the containerization meetings he has expressed the need to change the name of a collection after installation via API.